src/avt/Plotter/vtk/vtkVisItCubeAxesActor.C

```diff
@@ -120,26 +124,17 @@ vtkVisItCubeAxesActor::vtkVisItCubeAxesActor()
   for (i = 0; i < 4; i++)
     {
     this->XAxes[i] = vtkVisItAxisActor::New();
-    this->XAxes[i]->SetTickVisibility(true);
     this->XAxes[i]->SetMinorTicksVisible(true);
```

Comment:
SetMinorTicksVisible should be removed as well, since it is called in SetVisibility.

Reply:
Thanks, I must have missed that one.